testing/kokoro/system_tests.sh

```diff
@@ -67,6 +64,8 @@ echo "Running tests in project $GOLANG_SAMPLES_PROJECT_ID";
 # it when exiting.
 trap "go clean -modcache; gimmeproj -project golang-samples-tests done $GOLANG_SAMPLES_PROJECT_ID" EXIT
 
+set +x
```

Comment:
What is the reason for switching this on and off?

Reply:
Don't want to accidentally print a secret environment variable. We don't have any right now, but we've had issues in the past.

Reply:
Ahh makes sense-- might want to note that in a comment so someone doesn't remove it in the future.